src/CookieStore.ts

```diff
@@ -126,12 +127,15 @@ class CookieStore {
 
     if (persistedCookies) {
       try {
-        const parsedCookies: [string, [string, any]] = JSON.parse(
+        const parsedCookies: [string, [string, CookieString][]][] = JSON.parse(
           persistedCookies,
         )
 
-        parsedCookies.forEach(([origin, cookie]) => {
-          this.store.set(origin, new Map(cookie))
+        parsedCookies.forEach(([origin, cookies]) => {
+          this.store.set(origin, new Map(cookies.map(([ token, { expires, ...cookie }]) => [
+            token,
+            expires === undefined ? cookie : { ...cookie, expires: new Date(expires) }
```

Comment:
Nitpick: `expires == null` would check for both `null` and `undefined`. Maybe that's a safer option in case the value is `null`? Can it potentially be `null`?

Reply:
The type definition of `Cookie` says it can only be undefined or a string. I hope that's correct.